src/modules/Dashboard/features/Respondents/Popups/RespondentsRemoveAccessPopup/RespondentAccessPopup.utils.tsx

```diff
@@ -113,15 +112,22 @@ export const getScreens = ({
   submitPassword,
   removeAccess,
   handlePopupClose,
-}: ScreensParams) => {
-  const getResultScreen = (getSuccessScreen: getScreen, getErrorScreen: getScreen, title: string) =>
+  callbackFunction,
+}: ScreensParams): Screen[] => {
+  const handleClose = () => {
+    callbackFunction();
```

Comment:
does it have a chance to be merged as one function instead of two?

Reply:
Re-fetch respondents on the last step of the Modal need to be done, but in other steps, the action needs to be only closed.